crates/forge/tests/it/isolate.rs

```diff
@@ -0,0 +1,13 @@
+//! Isolation tests.
+
+use crate::{config::*, test_helpers::TEST_DATA_DEFAULT};
+use foundry_test_utils::Filter;
+
+#[tokio::test(flavor = "multi_thread")]
+async fn test_isolate_record_gas() {
```

Comment:
Can possibly inlined in regular local tests, may be clear this way

Reply:
@klkvr / @Evalir 

Ended up running into no issues with the default, non-isolation mode (except the discussed accuracy).
Therefore the new standalone isolation test isn't strictly necessary anymore. I think it is useful however to keep it around for if we do want to run tests specifically in isolation mode. I can remove it if you would prefer that.

Reply:
Resolved, moved to `cheats` with a strict filter that can be expanded if more cheatcodes require to be tested explicitly in isolation mode as well.